### PR TITLE
CD: publish the 3 packages

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

**Review by peterjah — APPROVED** [review]

File: scripts/publish-release.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+source ./scripts/enable-git-ssh.sh
+set-git-ssh
+
+PACKAGE=$1
+echo $PACKAGE
```
Comment (gregLibert) [inline scripts/publish-release.sh:8]: Why ?

File: scripts/publish-release.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+case $PACKAGE in
+    tester|transformer|as)
+        if [[ "$PACKAGE" == "as" ]]; then
+            NPM_PACKAGE=@massalabs/as
+        else
+            NPM_PACKAGE=@massalabs/as-$PACKAGE
+            pushd $PACKAGE
```
Comment (gregLibert) [inline scripts/publish-release.sh:16]: why not using `cd ..` ? You are not poping after it

File: scripts/publish-release.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        fi
+        # remove husky install script
+        npm pkg delete scripts.prepare
+        npm publish --access public
+        ;;
+    *)
+        echo "Unknown package $PACKAGE"
+        exit 1;;
+esac
```
Comment (gregLibert) [inline scripts/publish-release.sh:25]: You can improve readibility by removing the nested if:
```suggestion
NPM_PACKAGE=@massalabs/as
case $PACKAGE in
    tester|transformer)
        NPM_PACKAGE=$NPM_PACKAGE-$PACKAGE 
        cd ..
        ;&
    as) 
        npm pkg delete scripts.prepare # remove husky install script
        npm publish --access public
        ;;
    *)
        echo "Unknown package $PACKAGE"
        exit 1
        ;;
esac
```

More info on ;& [here](https://www.gnu.org/software/bash/manual/html_node/Conditional-Constructs.html#index-case)

**Review by gregLibert — CHANGES_REQUESTED** [review]
:no_good: missing an unwanted log
:mag: code readibility

File: scripts/publish-release.sh (hunk 3 above)
Comment (Thykof) [reply scripts/publish-release.sh]: nice!

**Review by gregLibert — APPROVED** [review]
:clap: clean code!

**Pull request merged**